mappings/UofM.py

```diff
@@ -75,6 +70,40 @@ def formatIdentifiers(self):
                     return newISBNList
                 
         return self.record.identifiers
+    
+    def formatSubjects(self):
+        subjectList = []
+
+        if '|' in self.record.subjects:
+            subjectList = self.record.subjects.split('|')
+            newSubjectList = [f'{subject}||' for subject in subjectList] 
+            return newSubjectList
+        else:
+            subjectList.append(f'{self.record.subjects}||')
+            return subjectList
+    
+    def formatRights(self):
+        # Parse rights codes
+        rightsElements = self.record.rights.split('|') if self.record.rights else [''] * 5
+        rightsMetadata = rightsElements[0]
+        licenseMeta = ''
+        statementMeta = ''
```

Comment:
Are there cases where we want the rights to be `UofM|||`?

Reply:
No to reach the formatRights method, there will have to be a value in the Rights mapping so there won't be a case for UofM||||. To make this clearer, I'll delete those two lines and move this code block 

```
rightsObject = 'UofM|{}||{}|'.format(
                licenseMeta,
                statementMeta
            ) 

            return rightsObject
```

inside both conditional statements.